### PR TITLE
kubevela 0.2.0(new formula)

### PR DESCRIPTION
**Pull request by sunny0826**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/kubevela.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class Kubevela < Formula
+  desc "Application Platform based on Kubernetes and Open Application Model"
+  homepage "https://kubevela.io"
+  url "https://github.com/oam-dev/kubevela.git",
+    using:    :git,
```
Comment (SMillerDev) [inline Formula/kubevela.rb:5]: This isn't needed

File: Formula/kubevela.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    tag:      "v0.2.0",
+    revision: "79c2209d19ddc62bb6682c91b3a485b4f939ab5f"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "git"
```
Comment (SMillerDev) [inline Formula/kubevela.rb:11]: It's this needed by the application? Can't it use the macOS one?

Comment (sunny0826) [reply Formula/kubevela.rb:11]: sorry

File: Formula/kubevela.rb (hunk 1 above)
Comment (sunny0826) [reply Formula/kubevela.rb]: get it

**Review by dawidd6 — DISMISSED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@sunny0826, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**